internal/cluster/cluster.go

```diff
@@ -20,54 +27,69 @@ func InstallHelm() {
 	cmd.Run()
 }
 
-func InitTerraform() {
-	currentDir := util.CurrentExecutableDir()
-	initScriptLocation := currentDir + "/terraform"
-	initScript := "./init.sh"
-
-	cmd := commandline.NewWithWorkingDir(initScript, []string{}, initScriptLocation)
-	cmd.Run()
+func InitTerraform(tf string, project string, env string) {
+	// Create Bucket if not present in the system
+	err := gcp.CreateGCSBucket(project, env)
+	if err != nil {
+		log.Fatal(err)
+	}
+	// Check if terraform initialized
+	if _, err := os.Stat(tf+"/.terraform"); os.IsNotExist(err) {
+		fmt.Printf("Terraform not Initialized: in %v: Intializong now...", tf+"/.terraform\n")
+		argsArray := util.StringTemplateToArgsArray("%s", "init")
+		cmd := commandline.NewWithWorkingDir("terraform", argsArray, tf)
```

Comment:
Use better `commandline.NewCommandLine(string)` which receives command as string

Reply:
That might not have `workingDir` actually, need another constructor :D

Reply:
Yup. I'll keep it for now. There is more code-rewrite coming up soon. For now, I need operational repo.